.drone.yml

```diff
@@ -139,4 +104,4 @@ trigger:
 
 depends_on:
 - linux-arm64
-# - linux-arm
+
```

Comment:
tphoney:arm64_build

Reply:
I am not sure what you are trying to tell me ?
the drone.yml is valid and builds are running.